website/docs/guides/microsoft-fabric-qs.md

```diff
@@ -108,6 +108,9 @@ A public preview of Microsoft Fabric in dbt Cloud is now available!
 2. Enter a project name and click **Continue**.
```

Comment:
Not super important, but to fix the code syntax in GitHub, you can add a space before ` ``` ` on line 101 (the code is light blue but should have syntax highlighting)